src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,30 @@
+.catalog {
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
```

Comment:
If I delete one card, the left cards shouldn't change its position
<img width="1012" alt="image" src="https://user-images.githubusercontent.com/32794096/201059940-ea5dd988-3140-4275-a135-0c725560fa54.png">